Flaresolverr error 400

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Flaresolverr

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/flaresolverr.sh)"

### 📝 Provide a clear and concise description of the issue.

When installing using all default setting (only turning on verbose mode to be able to report the error) the following error is returned:



### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Run the script
Choose Verbose mode to be able to see the details of the error.
Wait for the error to show up.

### ❌ Paste the full error output (if available).

  ✔️  Installed Chrome
 ⠹curl: (6) Could not resolve host: flaresolverr_linux_x64.tar.gz
 ⠸
[ERROR] in line 32: exit code 0: while executing command $STD curl -fsSL "https://github.com/FlareSolverr/FlareSolverr/releases/download/$RELEASE/flaresolverr_linux_x64.tar.gz" -O "flaresolverr_linux_x64.tar.gz"
curl: (22) The requested URL returned error: 400

### 🖼️ Additional context (optional).

_No response_

--- a/install/flaresolverr-install.sh
+++ b/install/flaresolverr-install.sh
@@ -29,10 +29,10 @@ msg_ok "Installed Chrome"
 
 msg_info "Installing FlareSolverr"
 RELEASE=$(curl -fsSL https://github.com/FlareSolverr/FlareSolverr/releases/latest | grep "title>Release" | cut -d " " -f 4)
-$STD curl -fsSL "https://github.com/FlareSolverr/FlareSolverr/releases/download/$RELEASE/flaresolverr_linux_x64.tar.gz" -O "flaresolverr_linux_x64.tar.gz"
+$STD curl -fsSL "https://github.com/FlareSolverr/FlareSolverr/releases/download/$RELEASE/flaresolverr_linux_x64.tar.gz" -o "flaresolverr_linux_x64.tar.gz"
 $STD tar -xzf flaresolverr_linux_x64.tar.gz -C /opt
 $STD rm flaresolverr_linux_x64.tar.gz
-echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
+echo "${RELEASE}" >/opt/"${APPLICATION}"_version.txt
 msg_ok "Installed FlareSolverr"
 
 msg_info "Creating Service"

